keyboards/keyap/info.json

```diff
@@ -0,0 +1,46 @@
+{
+    "manufacturer": "PenEngineering",
+    "keyboard_name": "Keyapad",
+    "maintainer": "PenEngineering",
+    "bootloader": "atmel-dfu",
```

Comment:
```suggestion
    "development_board": "elite_c",
```

Reply:
why????

Reply:
I use the atmega32u4 chip

Reply:
not necessarily dev board

Reply:
TQFP

Reply:
Well, you need some sort of microcontroller specified. If you're not willing to make this change, I suggest you update the readme accordingly and replace the microcontroller as `atmega32u4`.

Reply:
okey